optimade/server/routers/links.py

```diff
@@ -19,14 +21,12 @@
 
 @router.get(
     "/links",
-    response_model=LinksResponse,
+    response_model=LinksResponse if CONFIG.validate_api_response else Dict,
     response_model_exclude_unset=True,
     tags=["Links"],
     responses=ERROR_RESPONSES,
 )
-def get_links(
-    request: Request, params: EntryListingQueryParams = Depends()
-) -> LinksResponse:
+def get_links(request: Request, params: EntryListingQueryParams = Depends()) -> Any:
```

Comment:
Does the get_entries function not always return a dict ? And should therefore the get_links function not always return a dict as well ?

Reply:
This is where the `route()` decorator changes things. The FastAPI docs recommend using `Any` in this case to avoid mypy  errors.

Reply:
Ok, thank you for the explanation.